CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ For details about compatibility between different releases, see the **Commitment
 ### Fixed
 
 - Resolve scroll jumps when selecting different tabs of a table in the Console.
+- `BatchGetGatewayConnectionStats` RPC rights check in certain cases.
```

Comment:
Probably worth mentioning that the error with `duplicate gateway identifiers` on batch operations no longer happens.

Reply:
It's the same rights error check from the perspective of the caller (we say 'insufficient rights' because you queried the same gateway twice but the store implementation returned only one gateway).

Reply:
I agree, but at least on my mind, the reader would probably like to have a more specific example of what the fix entails

Reply:
Then again, the `BatchGetGatewayConnectionStats` probably gives enough context.

Disregard my comment then.